cscore/src/main/java/edu/wpi/cscore/MjpegServer.java

```diff
@@ -1,5 +1,5 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2016-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2016-2020 FIRST. All Rights Reserved.                        */
```

Comment:
Revert this change.

Reply:
The linter was flagging it on my fork, causing CI to fail. Is the formatter a non-gating check?